bindings/node.js/binding.gyp

```diff
@@ -0,0 +1,51 @@
+{
+  'targets': [
+    {
+      'target_name': 'blst-node-binding',
+      'sources': [
+        'blst_wrap.cpp',
+        '../libblst.a'
+      ],
+      # Is 'libblst.a' a library or a source?
+      'libraries': [],
```

Comment:
libblst.a is certainly a library. There is a caveat trough. It most likely have to be named differently on Windows. In other words this might have to be OS-conditional...